k6-extension/Makefile

```diff
@@ -0,0 +1,12 @@
+DOCKER_BUILD=docker build
```

Comment:
Can we add documentation on this Makefile, like a command `make help` like [here](https://github.com/kubeshop/tracetest/blob/main/Makefile#L8)?

Reply:
I stripped it down so it only has the things we need